frontend/src/pages/modelServing/ModelServingContext.tsx

```diff
@@ -44,12 +50,15 @@ export const ModelServingContext = React.createContext<ModelServingContextType>(
   servingRuntimeTemplateDisablement: DEFAULT_CONTEXT_DATA,
   servingRuntimes: DEFAULT_CONTEXT_DATA,
   inferenceServices: DEFAULT_CONTEXT_DATA,
+  project: null,
 });
 
-const ModelServingContextProvider: React.FC = () => {
+const ModelServingContextProvider = conditionalArea<ModelServingContextProviderProps>(
+  SupportedArea.MODEL_SERVING,
+  true,
+)(({ children, namespace }) => {
```

Comment:
Add an area checker for the global model serving page just like the pipelines page.